Octokit.Tests.Conventions/ModelTests.cs

```diff
@@ -85,6 +85,12 @@ public void ResponseModelsHaveReadOnlyCollections(Type modelType)
                         continue;
                     }
 
+                    // Let's skip collections that only have a private Setter
+                    if (property.SetMethod != null && property.SetMethod.IsPrivate)
```

Comment:
But this is totally out of scope for this test, and it actually makes it incorrect. This test checks that "Response Models Have Read Only Collections". Making it skip properties with private setters also makes it allow mutable collections, which defeats the test in the first place. There's already [a test that checks for non-public setters](https://github.com/octokit/octokit.net/blob/master/Octokit.Tests.Conventions/ModelTests.cs#L42-L62).